libraries/resource_firewall.rb

```diff
@@ -19,5 +19,11 @@ class Resource::Firewall < Chef::Resource::LWRPBase
 
     # for firewall implementations where ipv6 can be skipped (currently iptables-specific)
     attribute(:ipv6_enabled, kind_of: [TrueClass, FalseClass], default: true)
+
+    if Chef::Platform.windows?
```

Comment:
I did not get  your point. This allows to define attribute if it is running on Windows platform? 

Can you please explain more what you meant?